cmd/deploy/run.go

```diff
@@ -1,13 +1,83 @@
 package deploy
 
 import (
-	"fmt"
-
+	"github.com/0xSplits/kayron/pkg/cache"
+	"github.com/0xSplits/kayron/pkg/envvar"
+	"github.com/0xSplits/kayron/pkg/operator"
+	"github.com/0xSplits/kayron/pkg/operator/policy"
+	"github.com/0xSplits/kayron/pkg/runtime"
+	"github.com/0xSplits/otelgo/recorder"
+	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/spf13/cobra"
+	"github.com/xh3b4sd/logger"
+	"github.com/xh3b4sd/tracer"
+	"go.opentelemetry.io/otel/metric"
 )
 
-type run struct{}
+type run struct {
+	flag *flag
+}
+
+func (r *run) runE(cmd *cobra.Command, arg []string) error {
+	var env envvar.Env
+	{
+		env = envvar.Load(r.flag.Env)
+	}
```

Comment:
All of this logic is basically the same code that we also use in our integration test.